Common/cpp/SharedItems/Shareables.h

```diff
@@ -0,0 +1,249 @@
+#pragma once
+
+#include <jsi/jsi.h>
+#include <memory>
+
+#include "RuntimeManager.h"
+#include "Scheduler.h"
+
+using namespace facebook;
+
+namespace reanimated {
+
+class CoreFunction;
+
+class JSRuntimeHelper {
```

Comment:
This name seems to be too general, is it some kind of runtime scheduler?

Reply:
it was meant to be that way. We already had a number of similar helpers and I was hoping to unify them into a single class after all the code that can go away is removed. Need to revisit this topic now and will make updates accordingly